pkg/rules/rules_test.go

```diff
@@ -82,7 +82,7 @@ func testRulesAgainstExamples(t *testing.T, dir string, server rulespb.RulesServ
 		{
 			Name:                    "thanos-sidecar",
 			File:                    filepath.Join(dir, "alerts.yaml"),
-			Rules:                   []*rulespb.Rule{someAlert, someAlert, someAlert},
```

Comment:
Do we  need this change?

Reply:
Since we have removed the `ThanosSidecarPrometheusDown` alert, now sidecar can emit only 2 alerts, hence the change :)